api/server/controllers/meetingRequestController.js

```diff
@@ -0,0 +1,202 @@
+MeetingRequest = require('../models/meetingRequestModel')
+
+// Create a new meeting request
+exports.create = function(req, res) {
+    let meetingRequest = new MeetingRequest();
+    meetingRequest.author = req.params.author;
+    meetingRequest.deadline = req.params.deadline;
+    meetingRequest.groupId = req.params.groupId;
+    meetingRequest.multiPoll = req.params.multipoll ? req.params.multipoll : [];
+    meetingRequest.uniPoll = req.params.uniPoll;
+    meetingRequest.staus = req.params.status;
+    meetingRequest.isUniPoll = req.params.isUniPoll;
```

Comment:
If we have this flag here, then instead of having the 'multiPoll' and 'uniPoll' attributes, we can just have an attribute called options that is an array of Option's. For unanimous polls, just put one option in the array. 

Also, I think you can just spell the whole thing out: 'isUnanimous' or 'isPoll' instead of 'isUniPoll', since 'Uni' is unclear and unanimous meeting requests aren't actually 'polls'.

Reply:
Makes sense. I just kept the naming scheme from earlier because I thought that's what everyone wanted